packages/extension-chakra-storefront/src/theme/components/base/button.js

```diff
@@ -41,12 +49,14 @@ const button = defineRecipe({
                 paddingLeft: 3,
                 paddingRight: 3
             },
-            // link: (props) => ({
-            //     color: props.colorPalette === 'red' ? 'red.500' : 'blue.600',
-            //     fontWeight: 'normal',
-            //     minWidth: '1em',
-            //     lineHeight: 4
-            // }),
+            link: {
+                ...baseLinkStyles,
+                color: 'blue.600'
+            },
+            'error-link': {
+                ...baseLinkStyles,
+                color: 'red.500'
+            },
```

Comment:
For this button theme,  look at the code changes in #2555 as it's more complete there. For example, it has the styles for the hovered state.

Reply:
I made a hybrid because I saw there was a lot of duplicate styles in your example, but added the hover and other styles you added to my shared object.